src/static/commandline.css

```diff
@@ -9,7 +9,6 @@ input {
   font-family: monospace;
```

Comment:
You don't need to modify this file: default colors and so on can live in here and be overridden by a dark theme.